services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -287,6 +287,9 @@ private ClientRepresentation modelToRepresentation(ClientModel model, List<Strin
         UserConsentModel consentModel = consents.get(model.getClientId());
         if(consentModel != null) {
             representation.setConsent(modelToRepresentation(consentModel));
+            representation.setLogoUri(model.getAttribute("logoUri"));
```

Comment:
Does it make sense to use the "config wrapper" to fetch those new properties rather than using their names directly (considering you have created constants for them)?